### PR TITLE
Terms Addition

### PR DESCRIPTION
**Pull request by jhhuynh42**
Created a terms and conddition and put footer link to it.

### CODE REVIEW

File: modules/orderService.js [hunk 1]
```diff
@@ -89,7 +89,7 @@ module.exports.addOrder = (newSID, newAdd, newStatus, newTotal) => {
 
 module.exports.UpdateOrder = (OrdId, newStatus) => {
 	return new Promise((resolve, reject) => {
-		Products.updateOne({ _id: OrdId }, { status: newStatus }, (err, result) => {
+		Products.updateOne({ _id: OrdId }, { status: newStatus, updated_at: Date.now }, (err, result) => {
```
Comment (lucacataldo) [inline modules/orderService.js]: I could be wrong, but Date.now is just the reference to the function now(). Probably want to change this to Date.now()

Comment (lucacataldo) [reply modules/orderService.js]: image

File: views/terms.hbs [hunk 2]
```diff
@@ -0,0 +1,108 @@
+<div class="container-fullwidth land-splash">
```
Comment (lucacataldo) [inline views/terms.hbs]: classes like land-splash shouldn't be used here, they're only for the landing page. It includes undesired behavior like scroll jumping and a coloured background which makes it hard to read.

File: modules/orderService.js (hunk 1 above)
Comment (jhhuynh42) [reply modules/orderService.js]: Date.now is how it's declared.

File: views/terms.hbs (hunk 2 above)
Comment (jhhuynh42) [reply views/terms.hbs]: Also is there a way to deal with navbar blocking the top of the container?

**Pull request closed without merging**